src/EpisodeIntegrationService/ReceiveData/ReceiveData.cs

```diff
@@ -179,24 +183,65 @@ private EpisodeDto MapEpisodeToEpisodeDto(BssEpisode episode)
         }
     }
 
-    private async Task ProcessParticipantDataAsync(IEnumerable<Participant> participants, string participantUrl)
+    private async Task ProcessParticipantDataAsync(string name,IEnumerable<Participant> participants, string participantUrl)
     {
+
+        DateTime processingStart = DateTime.UtcNow;
+
+        int successCount = 0;
+        int failureCount = 0;
+        int lastProcessedRow = 0;
+        int rowIndex = 0;
+
         try
         {
-            _logger.LogInformation("Processing participant data.");
+            _logger.LogInformation("Processing started for file: {name} at {processingStart}",name, processingStart);
+
             foreach (var participant in participants)
             {
                 string serializedParticipant = JsonSerializer.Serialize(participant, new JsonSerializerOptions { WriteIndented = true });
 
                 _logger.LogInformation("Sending participant to {Url}: {Request}", participantUrl, serializedParticipant);
 
                 await _httpRequestService.SendPost(participantUrl, serializedParticipant);
+
+                successCount++;
+                rowIndex++;
+                lastProcessedRow = rowIndex;
+
+                _logger.LogInformation("Row of No.{lastProcessedRow} processed successfully",lastProcessedRow);
             }
+
+
         }
         catch (Exception ex)
         {
             _logger.LogError(ex, "Error in ProcessParticipantDataAsync: {Message}", ex.Message);
-            await ProcessParticipantDataAsync(participants, participantUrl);
+
+            failureCount++;
+            rowIndex++;
+            lastProcessedRow = rowIndex;
```

Comment:
The rowIndex variable seems redundant. Instead of incrementing rowIndex and then assigning it to lastProcessedRow, why not just increment lastProcessedRow and remove the rowIndex variable?